packages/storage/src/implementation/string.ts

```diff
@@ -28,7 +29,7 @@ export const StringFormat = {
 };
 
 /**
- * @struct
+ * @internal
```

Comment:
Again, I think we could drop this since this is in an internal source folder.

Reply:
Removed.